content/applications/inventory_and_mrp/inventory/management/reporting/inventory_valuation_config.rst

```diff
@@ -82,27 +82,184 @@ Costing method
+
+      .. list-table::
+         :header-rows: 1
+         :stub-columns: 1
+
+         * - Operation
+           - Unit Cost
+           - Qty On Hand
+           - Incoming Value
+           - Inventory Value
+         * -
+           - $10
+           - 0
+           -
+           - $0
+         * - Receive 8 products for $10/unit
+           - $10
+           - 8
+           - 8 * $10
+           - $80
+         * - Receive 4 products for $16/unit
+           - $10
+           - 12
+           - 4 * $10
+           - $120
+         * - Deliver 10 products
+           - $10
+           - 2
+           - -10 * $10
+           - $20
+         * - Receive 2 products for $9/unit
+           - $10
+           - 4
+           - 2 * $10
+           - $40
+
+   .. tab:: Average Cost (AVCO)
+
+      Calculates the valuation of a product based on the average cost of that product, divided by
+      the total number of available stock on-hand. With this costing method, inventory valuation is
+      *dynamic*, and constantly adjusts based on the purchase price of products.
+
+      .. list-table::
+         :header-rows: 1
+         :stub-columns: 1
+
+         * - Operation
+           - Unit Cost
+           - Qty On Hand
+           - Incoming Value
+           - Inventory Value
+         * -
+           - $0
+           - 0
+           -
+           - $0
+         * - Receive 8 products for $10/unit
+           - $10
+           - 8
+           - 8 * $10
+           - $80
+         * - Receive 4 products for $16/unit
+           - $12
+           - 12
+           - 4 * $16
+           - $144
+         * - Deliver 10 products
+           - $12
+           - 2
+           - -10 * $12
+           - $24
+         * - Receive 2 products for $6/unit
+           - $9
+           - 4
+           - 2 * $6
+           - $36
+
+      .. spoiler:: How are Unit Cost and Inventory Value calculated at each step?
+
+         - When receiving four products for $16 each:
+
+           - **Inventory Value** is calculated by adding the previous inventory value with the
+             incoming value. So, with `$80 + (4 * $16)` = `$80 + $64` = `$144`.
+           - **Unit Cost** is calculated by dividing the inventory value by the quantity on hand.
+             So, `$144 / 12 = $12`.
+
+         - When delivering ten products, the average **Unit Cost** is used to calculate the
+           **Inventory Value**, regardless of the purchase price of the product. Therefore,
+           inventory value is `$144 + (-10 * $12)` = `$144 - $120` = `$24`
+
+         - Receive two products for $6 each:
+
+           - **Inventory Value**: `$24 + (2 * $6)` = `$24 + $12` = `$36`
+           - **Unit Cost**: `$36 / 4 = $9`
+
+      .. note::
+         When choosing :guilabel:`Average Cost (AVCO)` as the :guilabel:`Costing Method`, changing
+         the numerical value in the :guilabel:`Cost` field for products in the respective product
```

Comment:
```suggestion
         the numerical value in the *Cost* field for products in the respective product
```
opting for italics here because, since the reader/user is on a Product Category form, and the 'Cost' field on a product form isn't currently being shown on the UI/screen at this moment.